### PR TITLE
✨(tests) add section customization

### PR DESCRIPTION
**Pull request by Kitero**
## Purpose

Edition title and description on sections

### CODE REVIEW

File: src/components/Fields/Section.tsx [hunk 1]
```diff
@@ -1,14 +1,38 @@
...
   return (
     <fieldset>
       <legend>{title}</legend>
-      {isEditor && <DropZone id={idSchema['$id']} />}
+      <div>{description}</div>
+      <button onClick={() => setIsEditing(!isEditing)}>
+        <FormattedMessage {...messages.sectionParameters} />
+      </button>
+      {isEditing && <SectionPropertiesForm id={idSchema.$id} onClose={() => setIsEditing(false)} />}
+      {isEditor && !isRoot && !selector && <DropZone id={idSchema['$id']} />}
```
[the remark this answers is not in the document]
Comment (Kitero) [reply src/components/Fields/Section.tsx]: I added a comment

File: src/components/Fields/Section.tsx [hunk 2]
```diff
@@ -1,14 +1,43 @@
 import type { ObjectFieldTemplateProps } from '@rjsf/core';
 import DropZone from '../EditorFieldTemplate/DropZone';
 import { useVerna } from '../../providers/VernaProvider';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import React, { useState } from 'react';
+import SectionPropertiesForm from '../PropertiesForms/SectionPropertiesForm';
```
Comment (jbpenrath) [inline src/components/Fields/Section.tsx:6]: Sort import properly

File: src/components/PropertyForms/SectionPropertyForm.tsx [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+  },
+  legend: {
+    defaultMessage: 'Options',
+    description: 'Legend of the section',
+    id: 'components.SectionPropertiesForm.legend',
+  },
+  legendDescription: {
+    defaultMessage: 'Parameters for this section',
+    description: 'Legend of the section',
+    id: 'components.SectionPropertiesForm.legend',
```
[the remark this answers is not in the document]
Comment (jbpenrath) [reply src/components/PropertyForms/SectionPropertyForm.tsx]: Those two keys are redundant nope ? The `description` message seems not to be used.
```
description: {
    defaultMessage: 'Description',
    description: 'Description of the section',
    id: 'components.SectionPropertiesForm.description',
  },
```

```
legendDescription: {
    defaultMessage: 'Parameters for this section',
    description: 'Description of the section',
    id: 'components.SectionPropertiesForm.legendDescription',
  },
```

**Review by jbpenrath — APPROVED** [review]
LGTM ✅

**Pull request merged**